src/server/mod.rs

```diff
@@ -53,6 +55,9 @@ pub fn run(config: Config) -> Fallible<()> {
     let bot_username = github.username()?;
     let acl = ACL::new(&config, &github)?;
 
+    let opts = prometheus::opts!("crater_completed_jobs", "completed jobs");
```

Comment:
Thinking about it, to be coherent with [Prometheus's naming practices](https://prometheus.io/docs/practices/naming/#metric-names) this should be named `crater_completed_jobs_total`.